torch_em/data/image_collection_dataset.py

```diff
@@ -95,11 +96,16 @@ def ndim(self):
         return self._ndim
 
     def _sample_bounding_box(self, shape):
+        if self.patch_shape is None:
+            patch_shape_for_bb = shape
+        else:
+            patch_shape_for_bb = self.patch_shape
```

Comment:
I would also include the `bb_start` logic in this `if-else` and set it to `[0]* len(shape)` if the patch shape is None. 
While I think the current logic woks, it is not very readable.

Reply:
Thanks!